clients/src/main/java/org/apache/kafka/common/metrics/stats/TotalCount.java

```diff
@@ -14,25 +14,18 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.kafka.streams.processor.internals.metrics;
+package org.apache.kafka.common.metrics.stats;
```

Comment:
This metric had been added as an internal class in Streams, since it was only needed by Streams, but this actually seems like a liability...

Putting it here reduces the chance that someone would accidentally use `Count`, not realizing it is sampled, whereas this metric's presence would force you to consider which of the two options you want.

Also, if someone did want a non-sampled count in the future, they might just re-implement it here, if they didn't happen to find the one in Streams, leading us back into the situation I'm trying to fix here, where we have redundant metrics.

These two points seem to outweigh the advantage of keeping a class defined in the smallest scope in which it's needed.